packages/terra-application-name/src/orion-fusion-theme/ApplicationMenuName.module.scss

```diff
@@ -0,0 +1,27 @@
+:local {
+  .orion-fusion-theme {
+    --terra-application-name-menu-border-bottom: 1px solid #a9a9a9;
+    --terra-application-name-menu-color: #fff;
+    --terra-application-name-menu-title-text-shadow: none;
```

Comment:
this is listed 2x
```suggestion

```
It might be easiest to list these var in the order they appear in the style sheet so it's quick to find their original usage / intention.